cmd/tckconvert.cpp

```diff
@@ -153,6 +192,7 @@ class VTKWriter: public WriterInterface<float> { MEMALIGN(VTKWriter)
 
   private:
     File::OFStream VTKout;
+    bool write_binary;
```

Comment:
I'd make that a `const bool` if that's OK.

Reply:
Sure, also swapping the sense to `write_ascii` and making `binary` the default, related to above comment.